daemon/cluster/executor/container/adapter.go

```diff
@@ -450,13 +444,11 @@ func (c *containerAdapter) logs(ctx context.Context, options api.LogSubscription
 		}
 	}
 
-	chStarted := make(chan struct{})
+	chMsgs := make(chan *logger.Message, 1)
 	go func() {
-		defer writer.Close()
-		c.backend.ContainerLogs(ctx, c.container.name(), apiOptions, chStarted)
+		c.backend.ContainerLogsRawChan(ctx, c.container.name(), apiOptions, chMsgs)
 	}()
```

Comment:
Could be just `go c.backend.ContainerLogsRawChan(ctx, c.container.name(), apiOptions, chMsgs)`